internal/decryption/pem/pem.go

```diff
@@ -54,8 +65,8 @@ func load(fs afero.Fs, filename string) (map[BlockType]*pem.Block, error) {
 	return pemBlocks, nil
 }
 
-func Decode(fs afero.Fs, filename, password string) (cert, privateKey []byte, err error) {
-	pemBlocks, err := load(fs, filename)
+func (p *PEM) Decode(filename, password string) (cert, privateKey []byte, err error) {
```

Comment:
save yourself the use of `Default()` in consumers
```suggestion
func (p *PEM) Decode(filename, password string) (cert, privateKey []byte, err error) {
	return defaultPem.Decode(filename, password)
}
func (p *PEM) Decode(filename, password string) (cert, privateKey []byte, err error) {
```